sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -334,6 +334,75 @@ export class NotificationActionsService {
     return `${getPSTPDTDateTime(date)} PST/PDT`;
   }
 
+  /**
+   * Create assessment ready for student confirmation notification to notify student
+   * when workflow update the NOA approval status.
+   * @param notification notification details.
+   * @param auditUserId user who updates NOA approval status.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  async saveNOAApprovalStatusNotification(
```

Comment:
I would recommend renaming the method to what the notification represents (e.g. AssessmentReadyForConfirmation) instead the moment that we are triggering it (NOA approval).

Reply:
okay... actually i was following the other create notification methods naming convention